src/main/java/org/codeforamerica/shiba/pages/PageController.java

```diff
@@ -233,9 +233,12 @@ ModelAndView getPage(
               landmarkPagesConfiguration.getLandingPages().get(0)));
     }
 
-    response.addHeader("Cache-Control", "no-store");
-
     var pageWorkflowConfig = applicationConfiguration.getPageWorkflow(pageName);
+    if (pageWorkflowConfig == null) {
+      return new ModelAndView("redirect:/error");
+    }
+
+    response.addHeader("Cache-Control", "no-store");
```

Comment:
@andrewedstrom Does this bust the cache?

Reply:
Not 100% sure. That line was already there, I just moved it down below this validation.